compel/test/infect/spy.c

```diff
@@ -176,12 +178,10 @@ int main(int argc, char **argv)
 	printf("Checking the result\n");
 
 	/* These two came from parasite */
-	pass = chk(p_out[0], 138);
-	pass = chk(p_out[0], 403);
+	pass = chk(p_out[0], 138) && chk(p_out[0], 403);
 
 	/* These two came from post-infect */
-	pass = chk(p_out[0], 1234);
-	pass = chk(p_out[0], 4096);
+	pass = pass && chk(p_out[0], 1234) && chk(p_out[0], 4096);
```

Comment:
May be shorter `pass &&= chk(p_out[0], 1234) && chk(p_out[0], 4096);` or even separate chk to different lines:
```
pass &&= chk(p_out[0], 1234);
pass &&= chk(p_out[0], 4096);
```

Reply:
Logical assignment operators don't exist in C.

(You were possibly thinking the bitwise `&=`?)

Reply:
> Logical assignment operators don't exist in C.

Yes, my mistake =)

I'd then advise to reverse return of chk() as it is looks cleaner like this https://github.com/Snorch/criu/commit/6c3214445dec26c8f3f2faef5bb1aa1740935b7a

Hope I don't miss something this time =)

Reply:
Thanks, applied in both tests.